gandy/quality_est/metrics.py

```diff
@@ -1,151 +1,155 @@
-'''Metrics module: contains some relevent metrics to assess the performance of machine learning models. 
-
-This module implements a parent metric class that contains necessary initialization arguments and automatically 
-calls a calculate method to compute a given metric. Required intial arguments include the machine learning model 
-output predictions and real values for comparison. Optionally, the user may input uncertainties if a given model 
-outputs them. The properties of the parent class are then inhereted by individual children classes which define 
-the exact mathematical operations to compute a specific metric. Calling a specific metric child class will compute 
-a given metric and return the total value and/or invidual values of that metric based on the input data provided. 
+'''Metrics module: contains some relevent metrics to assess the performance of machine learning models.
+
+This module implements a parent metric class that contains necessary initialization arguments and automatically
+calls a calculate method to compute a given metric. Required intial arguments include the machine learning model
+output predictions and real values for comparison. Optionally, the user may input uncertainties if a given model
+outputs them. The properties of the parent class are then inhereted by individual children classes which define
+the exact mathematical operations to compute a specific metric. Calling a specific metric child class will compute
+a given metric and return the total value and/or invidual values of that metric based on the input data provided.
 '''
 
-## Imports 
+# Imports
 from typing import Tuple, Iterable, Any, Object, Type, List
 
 import numpy as np
-import math 
+import math
 
-## Typing 
+# Typing
 Array = Type[numpy.ndarray]
 
-## Define dictionary of available metrics 
+# Define dictionary of available metrics
 Metric_codex = {}
 
-## Parent class for metric 
+# Parent class for metric
+
+
```

Comment:
You might want to include this comment in the doc string instead of above the class definition.